Readme.md

```diff
@@ -1,7 +1,15 @@
+
+# Welcome to TunnelTestEnv
+The TunnelTestEnv Powershell script creates a full test environment for Microsoft Tunnel. The environment includes a Linux VM tunnel gateway server, and a second Linux VM that runs containers for a proxy server, a DNS server, and a web server. By default, an Intune tenant is also created, with tunnel profiles.
```

Comment:
Reword from an "Intune tenant is also created" to something like "Intune tenant is configured". We rely on an existing tenant, we don't create a new one.